### PR TITLE
CAF-2276 Define swagger contract

### PR DESCRIPTION
**Pull request by mulhollandc**
New swagger UI project added. 
ColumnName added to API specification. 
Enumeration change for parameter type field in the API specification.

### CODE REVIEW

File: caf-audit-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Audit Web Service UI
+
+This Swagger UI project builds a documentation web application for the [CAF Audit Web Service](https://github.com/CAFAudit/audit-service/tree/develop/caf-audit-service). It allows users to see what operations the web service exposes as well as documentation relating to those operations and the web service in general.
```
Comment (glucy) [inline caf-audit-ui/README.md]: As mentioned URL does not currently exists.  But OK for now.

File: caf-audit-ui/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## Usage
+
+To get started, deploy the [web service docker container](https://github.com/CAFAudit/audit-service/tree/develop/caf-audit-management-service-container). The web service UI can then be accessed at the following URL:
```
Comment (glucy) [inline caf-audit-ui/README.md]: Obviously we don't have an Audit Web Service Container yet, however this URL is definitely wrong as we no longer have an "caf-audit-management-service-container".

Once this URL is update, this is probably OK once the Web Service becomes available.

**Review by glucy — APPROVED** [review]
See comments, and assuming the URL are correct once a Web Service is available this is probably OK.

**Comment by mulhollandc** [comment]
README has been dropped as web service URLs not yet available. The web service implementation ticket has been modified to add the README with working URLs to the caf-audit-ui project when done.

**Comment by glucy** [comment]
README.md to be added as part of https://jira.autonomy.com/browse/CAF-2274, therefore merging.

**Pull request merged**